rotkehlchen/tests/external_apis/test_cowswap.py

```diff
@@ -10,3 +12,12 @@ def test_get_data_from_database(database):
         database=database,
         chain=SupportedBlockchain.ETHEREUM,
     ).get_order_data('TEST_ORDER') == (12345678, 'market')
+
+
+@pytest.mark.vcr
```

Comment:
I can let the tests run here so you can see if you used cassettes fine but yeah probably simple mocking is simpler. Even though VCR is exactly that, mocking.